be/src/vec/functions/function_ip.h

```diff
@@ -137,13 +139,7 @@ static inline bool try_parse_ipv4(const char* pos, Int64& result_value) {
 
 template <IPConvertExceptionMode exception_mode, typename ToColumn>
 ColumnPtr convert_to_ipv4(ColumnPtr column, const PaddedPODArray<UInt8>* null_map = nullptr) {
-    const ColumnString* column_string = check_and_get_column<ColumnString>(column.get());
-
-    if (!column_string) {
-        throw Exception(ErrorCode::INVALID_ARGUMENT,
-                        "Illegal column {} of argument of function {}, expected String",
-                        column->get_name());
-    }
+    const auto* column_string = check_and_get_column<ColumnString>(column.get());
```

Comment:
If nullptr is not checked here, then may be using assert cast？

Reply:
good advise. fixed it.